mkdocs.yml

```diff
@@ -16,6 +16,7 @@ nav:
       - Out of Incubator: docs/20190708-external-dns-incubator.md
       - Code of Conduct: code-of-conduct.md
       - License: LICENSE.md
```

Comment:
It's clearly not related to the focus of this PR.
May I suggest to move the changes in this file into an other PR ?